cmd/kyma/deploy/cmd.go

```diff
@@ -62,7 +62,7 @@ func NewCmd(o *Options) *cobra.Command {
 	- Deploy a pull request, for example "kyma deploy --source=PR-9486"
 	- Deploy the local sources: "kyma deploy --source=local"`)
 	cobraCmd.Flags().StringVarP(&o.Domain, "domain", "d", "", "Custom domain used for installation.")
-	cobraCmd.Flags().BoolVar(&o.DryRun, "dry-run", false, "Render manifests only.")
+	cobraCmd.Flags().BoolVar(&o.DryRun, "dry-run", false, "Alpha feature: Does not actually deploy, but renders manifests only. The generated resources are not enough to apply Kyma using kubectl, since components having custom installation routines (e.g. Istio) are not included.")
```

Comment:
Sounds strange that the sentence starts with telling what it is not doing. How about that?

```suggestion
	cobraCmd.Flags().BoolVar(&o.DryRun, "dry-run", false, "Alpha feature: Renders the kubernetes manifests only without actualy applying them. Currently, the generated resources are not sufficient to actually apply Kyma to a cluster, since components having custom installation routines (e.g. Istio) are not included.")
```

Reply:
```suggestion
	cobraCmd.Flags().BoolVar(&o.DryRun, "dry-run", false, "Alpha feature: Renders the Kubernetes manifests without actually applying them. The generated resources are not sufficient to apply Kyma to a cluster, because components having custom installation routines (such as Istio) are not included.")
```